packages/terra-form-input/src/InputField.jsx

```diff
@@ -143,6 +149,7 @@ const InputField = (props) => {
     }
   }
 
+  const inputType = (type) || inputAttrs.type;
```

Comment:
You shouldn't need the `()` around `type`.

Reply:
[removed](https://github.com/cerner/terra-core/blob/8bdcebd9da82c7a46dd80a17f6c6b5a98fd38049/packages/terra-form-input/src/InputField.jsx#L152)